draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -107,9 +107,13 @@ all other definitions, please look up the SCHC specification
 - Downlink: LoRaWAN term for a frame transmitted by the network and
   received by the device.
 - FRMPayload: Application data in a LoRaWAN frame.
+- MSB: Most Significant Byte
 - OUI: Organisation Unique Identifier. IEEE assigned prefix for EUI.
 - RCS: Reassembly Check Sequence. Used to verify the integrity of the
   fragmentation-reassembly process.
+- RX: Device's reception window following an uplink.
+- RX1/RX2: LoRaWAN devices must open two RX windows following an uplink,
```

Comment:
LoRaWAN class-A end-devices

Reply:
Thank you for the review bu I am not sure to understand your comment: RX1 and RX2 are still opened after TX in class B and C.

Reply:
When an end-device is transmitting, it behaves in class-A, whatever its capabilities. RX1 and RX2 are only defined for Class A.
Class B reception windows are called ping slots
Class C reception window is called RXC.

Reply:
Ok. Does the following suggestion work for you ?
```
- RX: Class A device's reception window, following an uplink.
- RX1/RX2: LoRaWAN devices must open two RX windows following an uplink,
```

Reply:
Actually, I would go the opposite, as RX windows are not specific to class A (E.g: RXC):

```
- RX: device's reception window
- RX1/RX2: LoRaWAN class A end-devices open two RX windows following an uplink, called RX1 and RX2.
```